runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -607,7 +596,8 @@ static TR_YesNoMaybe isValue(TR::VPConstraint *constraint, TR_OpaqueClassBlock *
       }
 
    TR::VPResolvedClass *type = maybeUnresolvedType->asResolvedClass();
-   if (type == NULL)
+   if (type == NULL ||
+       type->isJavaLangObject(this))
       {
       return TR_maybe;
```

Comment:
I'm not sure I fully follow the question here, but I think it might be related to [your following comment](https://github.com/eclipse-openj9/openj9/pull/16490#discussion_r1054466896). I [addressed it as below](https://github.com/eclipse-openj9/openj9/pull/16490#discussion_r1060714338). Is [that change](https://github.com/eclipse-openj9/openj9/pull/16490#discussion_r1060714338) what you were referring here?

Reply:
Yes

Reply:
Addressed in 62fa29ac